customer.go

```diff
@@ -17,6 +17,7 @@ type CustomerParams struct {
 	TrialEnd      int64
 	DefaultSource string
 	Shipping      *CustomerShippingDetails
+	BusinessVatId string
```

Comment:
Would you mind renaming this to `BusinessVatID` just to be more in-line with other identifiers in the project? [For example](https://github.com/stripe/stripe-go/blob/adffbfe3d540c9cb8fd8d7962df88b6c86714583/discount/client.go#L31).

It maybe should be `BusinessVATID`, but that looks like we're already using `BusinessVatID` elsewhere in the project.